tests/validation/tests/v3_api/test_multi_cluster_app.py

```diff
@@ -11,153 +27,161 @@
                   "p_client2": None, "namespace2": None,
                   "cluster2": None, "project2": None}
 
-global_client = {"client": None, "cluster_count": False}
-ROLES = ["project-member"]
-WORDPRESS_TEMPLATE_VID_2112 = "cattle-global-data:library-wordpress-2.1.12"
-WORDPRESS_TEMPLATE_VID_2111 = "cattle-global-data:library-wordpress-2.1.11"
-MYSQL_TEMPLATE_VID_037 = "cattle-global-data:library-mysql-0.3.7"
-MYSQL_TEMPLATE_VID_038 = "cattle-global-data:library-mysql-0.3.8"
-GRAFANA_TEMPLATE_VID = "cattle-global-data:library-grafana-0.0.31"
-WORDPRESS_EXTID = "catalog://?catalog=library&template=wordpress" \
-                  "&version=2.1.12"
-MYSQL_EXTERNALID_037 = "catalog://?catalog=library&template=mysql" \
-                       "&version=0.3.7"
-MYSQL_EXTERNALID_038 = "catalog://?catalog=library&template=mysql" \
-                       "&version=0.3.8"
-GRAFANA_EXTERNALID = "catalog://?catalog=library&template=grafana" \
-                     "&version=0.0.31"
-
-skip_race_condition = pytest.mark.skip(
-    reason='Test runs for a really long time, has to be looked into')
+global_client = {"cluster_count": False}
+PROJECT_ROLE = ["project-member"]
+CATALOG_URL = "https://github.com/rancher/integration-test-charts.git"
+BRANCH = "validation-tests"
+CATALOG_NAME = random_test_name("test-catalog")
+WORDPRESS_TEMPLATE_VID_738 = \
+    "cattle-global-data:" + CATALOG_NAME + "-wordpress-7.3.8"
+MYSQL_TEMPLATE_VID_131 = "cattle-global-data:" + CATALOG_NAME + "-mysql-1.3.1"
+MYSQL_TEMPLATE_VID_132 = "cattle-global-data:" + CATALOG_NAME + "-mysql-1.3.2"
+GRAFANA_TEMPLATE_VID = "cattle-global-data:" + CATALOG_NAME + "-grafana-3.8.6"
+WORDPRESS_EXTID = create_catalog_external_id(CATALOG_NAME,
+                                             "wordpress", "7.3.8")
+MYSQL_EXTERNALID_131 = create_catalog_external_id(CATALOG_NAME,
+                                                  "mysql", "1.3.1")
+MYSQL_EXTERNALID_132 = create_catalog_external_id(CATALOG_NAME,
+                                                  "mysql", "1.3.2")
+GRAFANA_EXTERNALID = create_catalog_external_id(CATALOG_NAME,
+                                                "grafana", "3.8.6")
+ROLLING_UPGRADE_STRATEGY = {
+        'rollingUpdate': {
+            'batchSize': 1,
+            'interval': 20,
+            'type': '/v3/schemas/rollingUpdate'},
+        'type': '/v3/schemas/upgradeStrategy'}
+
+skip_test_rolling_update = pytest.mark.skipif(
+    reason="timing issue.")
```

Comment:
the skip will always be true, so the function `def test_multi_cluster_rolling_upgrade():` will be skipped all the time, right?  Is it expected?

Reply:
added comments